### DISCUSSION TITLE
popper-append-to-body(deprecated) 被弃用怎么办

### DISCUSSION THREAD

**Discussion by 2512746832** [element-plus/element-plus#6022] [Q&A] [answered]
select 组件 的属性 popper-append-to-body(deprecated)   被弃用怎么办

**Comment by msidolphin** [comment, marked as answer]
teleported

**Reply by 2512746832** [reply]
element plus  现在popper-append-to-body 就不好用 ，我还以为已经弃用了

**Reply by wanfuxiong** [reply]
那如果是el-sub-menu中的popper-append-to-body呢
